tests/functional/disaster-recovery/metro-dr/test_multiple_apps_failover_and_relocate.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import pytest
+
+from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework import config
+
+from ocs_ci.ocs.acm.acm import AcmAddClusters
+from ocs_ci.ocs import constants
+from ocs_ci.helpers.dr_helpers import (
+    failover,
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    set_current_primary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    wait_for_all_resources_creation,
+    gracefully_reboot_ocp_nodes,
+)
+from ocs_ci.helpers.dr_helpers_ui import (
+    failover_relocate_ui,
+    verify_failover_relocate_status_ui,
+)
+from ocs_ci.framework.pytest_customization.marks import turquoise_squad
+from ocs_ci.utility import version
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+@turquoise_squad
+class TestMultipleApplicationFailoverAndRelocate:
+    """
+    Test Failover and Relocate actions for application
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, dr_workloads_on_managed_clusters):
+        """
+        If fenced, unfence the cluster and reboot nodes
+        """
+
+        def finalizer():
+
+            if (
+                self.primary_cluster_name is not None
+                and get_fence_state(self.primary_cluster_name) == "Fenced"
+            ):
+                enable_unfence(self.primary_cluster_name)
+                gracefully_reboot_ocp_nodes(self.primary_cluster_name)
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.parametrize(
+        argnames=["workload_type"],
+        argvalues=[
+            pytest.param(
+                constants.SUBSCRIPTION,
+            ),
+        ],
+    )
+    def test_application_failover_and_relocate(
+        self,
+        setup_acm_ui,
+        nodes_multicluster,
+        dr_workloads_on_managed_clusters,
+        workload_type,
+        node_restart_teardown,
+    ):
+        """
+        Tests to failover and relocate of applications from both the managed clusters
+
+        """
+        acm_obj = AcmAddClusters()
+        self.workload_namespaces = []
+        if config.RUN.get("mdr_failover_via_ui"):
```

Comment:
What will happen if yaml is not passed?

Reply:
It will run via CLI

Reply:
Sorry, I didn't get the else part. Could you please redirect me to the exact line?

Reply:
removed the check